### PR TITLE
Adding Dockship.io

### PR DESCRIPTION
**Pull request by ToluClassics**
Dockship allows you to contribute Machine Learning and Data Science articles for a fee for every upvote and views

### CODE REVIEW

**Comment by malgamves** [comment]
hey @ToluClassics I can't seem to find the application link on the web page you added. Could you change that??

**Comment by ToluClassics** [comment]
Hi @malgamves, you need to create a profile to be able to write.
Screenshot 2021-03-26 at 09 06 09

File: README.md [hunk 1]
```diff
@@ -83,6 +86,9 @@ A list of companies that have paid Developer Community Writer Programs.
 
 - [Rancher](https://rancher.com/writing-program/roles/writer/) - $300 per piece
   > Writing about devops, Kubernetes, and Rancher.
+ 
+- [Paperspace](https://blog.paperspace.com/write-for-paperspace/) - $200-$300 per piece
```
[the remark this answers is not in the document]
Comment (ToluClassics) [reply README.md]: Oh, Thanks for pointing that out. fixed it!

**Comment by ToluClassics** [comment]
Hi @malgamves, Made the updates

**Pull request merged**